### PR TITLE
Movable video frames

### PR DESCRIPTION
**Pull request by kaszim**
Added a way to switch places of videoframes. To do this one has to press ctrl+mouse1 on a video frame to select it. Then press ctrl+mouse1 on another one to switch places.

What is missing:
- [x] There is not indication that a frame has been selected 
- [ ] Beautify the code (do we really want more code in main? input wanted)

### CODE REVIEW

**Comment by kaszim** [comment]
I added a hue. However I had to add a constant. There is probably some better way to do it. If you do know a way, please let me know,

File: constants.py [hunk 1]
```diff
@@ -7,3 +7,9 @@
 ADD_NEW_STREAM = 'add_new_stream'
 EXPORT_STREAMS_TO_CLIPBOARD = 'export_streams_to_clipboard'
 SLIDER_MAX_VALUE = 10000 # The maxvalue in the ui for the rewind slider
+FRAME_SELECT_STYLE = """QFrame
+                        {
+                            border-style: outset;
+                            border-width: 2px;
+                            border-color: blue;
+                        }"""
```
Comment (mictab) [inline constants.py]: Could you add a newline to the file?

Comment (kaszim) [inline constants.py]: Good catch 👍

**Comment by mictab** [comment]
LGTM. Let's do the refactoring later. #refactoring-hackathon wooo 🎉

**Pull request merged**